src/core/infer_request.cc

```diff
@@ -425,6 +425,9 @@ InferenceRequest::PrepareForInference()
 Status
 InferenceRequest::Normalize()
 {
+  if (priority_ == 0) {
```

Comment:
This is taken care of in SetPriority

Reply:
SetPriority will not be called if the request doesn't specify a priority

Reply:
Then I think it would be better to call SetPriority(0) from the InferRequest constructor so that the logic stays in 1 place.